_lifeofinternatpuppet/post_42.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: Week 42
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 42
+---
+
+Hey everyone,
+
+A reply was sent to the IT Support in the ticket raised last week after chatting to my mentor Paula in my weekly 1:1 session of what to do with the 'unapproved process page' archived in the Puppet Forge Space on Confluence.
+
+I had many follow-up meetings with my manager this week on the Confluence Wiki Revamp Project. I tried to learn how to break down the bigger task into smaller sub tasks in JIRA so that the are easy to manage. Currently, I am struggling to write good, simple and easy to understand JIRA tickets with relevant sub tasks due to which my work is not progressing too well. I am thinking of how to overcome this new hurdle!
```

Comment:
Would perhaps rephrase a bit:

```suggestion
I had many follow-up meetings with my manager this week on the Confluence Wiki Revamp Project. I tried to learn how to break down the bigger task into smaller sub tasks in JIRA so that the are easy to manage. I was given feedback on how to improve on this process based on the previous attempts - I am thinking of how to overcome this new hurdle!
```

Reply:
I would dismiss it and keep it the way it is!